Framework/API/inc/MantidAPI/AlgorithmFactoryObserver.h

```diff
@@ -0,0 +1,61 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_KERNEL_ALGORITHMFACTORYOBSERVER_H_
+#define MANTID_KERNEL_ALGORITHMFACTORYOBSERVER_H_
+
+#include "MantidAPI/AlgorithmFactory.h"
+#include "MantidAPI/DllConfig.h"
+#include "MantidKernel/DataService.h"
```

Comment:
Is this include required?

Reply:
DataService was an accidental include from copying over file boilerplate. I assume DllConfig is too but I'm not sure what that does?

Reply:
`DllConfig` defines the `MANTID_API_DLL` that exports the symbol form the DLL. It will be included from the `AlgorithmFactory.h`.